### PR TITLE
chore: Replace dashboard icon with Material Symbols

### PR DESCRIPTION
**Pull request by susnux**
## Summary

Currently we use a circle as the dashboard icon,
1. This confuses with the circles app
2. Does not really gives information what this app does
3. Is not consistent with our app stores

So make it consistent and use the "dashboard" Material Symbol everywhere.

### Screenshot
Bildschirmfoto_20240701_120954

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by szaimen — APPROVED** [review]
Good idea!

**Review by AndyScherzinger — APPROVED** [review]

File: core/img/categories/dashboard.svg.license [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<!--
+ - SPDX-FileCopyrightText: 2018-2024 Google LLC
+ - SPDX-License-Identifier: Apache-2.0
+ - Material Symbols: dashboard
+-->
```
Comment (AndyScherzinger) [inline core/img/categories/dashboard.svg.license:5]: would simply do 
```
SPDX-FileCopyrightText: 2018-2024 Google LLC
SPDX-License-Identifier: Apache-2.0

```

Comment (susnux) [reply core/img/categories/dashboard.svg.license]: done

**Review by AndyScherzinger — APPROVED** [review]

**Review by jancborchardt — COMMENTED** [review]
Just FYI, the circle is a reference to the Nextcloud logo, and as the dashboard being the center of your stuff.

Yes the new dashboard icon is a more visual representation of what it is / how it looks like. Either is fine with me.

**Review by marcoambrosini — APPROVED** [review]

**Comment by susnux** [comment]
> Just FYI, the circle is a reference to the Nextcloud logo, and as the dashboard being the center of your stuff.

I was told that and some other places, but I think you do not notice it anymore as there are too many apps.
But this is why I requested reviews of all @nextcloud/designers as this is purely a design decision :)

**Pull request merged**